2021-12-13 17:51:48,500 INFO    MainThread:8932 [wandb_setup.py:_flush():71] setting env: {}
2021-12-13 17:51:48,501 INFO    MainThread:8932 [wandb_setup.py:_flush():71] setting login settings: {}
2021-12-13 17:51:48,501 INFO    MainThread:8932 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_175148-1me8snfi\logs\debug.log
2021-12-13 17:51:48,501 INFO    MainThread:8932 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\Usuario\Desktop\Uni\DataDaniMAPSIV\PSIV\wandb\run-20211213_175148-1me8snfi\logs\debug-internal.log
2021-12-13 17:51:48,503 INFO    MainThread:8932 [wandb_init.py:init():404] calling init triggers
2021-12-13 17:51:48,503 INFO    MainThread:8932 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-12-13 17:51:48,505 INFO    MainThread:8932 [wandb_init.py:init():449] starting backend
2021-12-13 17:51:48,505 INFO    MainThread:8932 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2021-12-13 17:51:48,509 INFO    MainThread:8932 [backend.py:ensure_launched():199] starting backend process...
2021-12-13 17:51:48,652 INFO    MainThread:8932 [backend.py:ensure_launched():204] started backend process with pid: 1112
2021-12-13 17:51:48,654 INFO    MainThread:8932 [wandb_init.py:init():458] backend started and connected
2021-12-13 17:51:48,656 INFO    MainThread:8932 [wandb_init.py:init():519] updated telemetry
2021-12-13 17:51:48,689 INFO    MainThread:8932 [wandb_init.py:init():549] communicating current version
2021-12-13 17:51:49,537 INFO    MainThread:8932 [wandb_init.py:init():554] got version response 
2021-12-13 17:51:49,537 INFO    MainThread:8932 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-12-13 17:51:49,789 INFO    MainThread:8932 [wandb_init.py:init():592] starting run threads in backend
2021-12-13 17:51:49,935 INFO    MainThread:8932 [wandb_run.py:_console_start():1816] atexit reg
2021-12-13 17:51:49,937 INFO    MainThread:8932 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2021-12-13 17:51:49,938 INFO    MainThread:8932 [wandb_run.py:_redirect():1727] Wrapping output streams.
2021-12-13 17:51:49,943 INFO    MainThread:8932 [wandb_run.py:_redirect():1751] Redirects installed.
2021-12-13 17:51:49,943 INFO    MainThread:8932 [wandb_init.py:init():619] run started, returning control to user process
2021-12-13 17:51:49,946 INFO    MainThread:8932 [wandb_config.py:__setitem__():144] config set learning_rate = 1e-05 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000027368FA5A60>>
2021-12-13 17:51:49,946 INFO    MainThread:8932 [wandb_run.py:_config_callback():962] config_cb learning_rate 1e-05 None
2021-12-13 17:51:49,947 INFO    MainThread:8932 [wandb_config.py:__setitem__():144] config set batch_size = 10 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x0000027368FA5A60>>
2021-12-13 17:51:49,947 INFO    MainThread:8932 [wandb_run.py:_config_callback():962] config_cb batch_size 10 None
2021-12-13 17:51:49,948 INFO    MainThread:8932 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-12-13 17:51:49,949 INFO    MainThread:8932 [wandb_run.py:_restore():1758] restore
2021-12-13 17:51:50,966 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 17:51:51,073 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 17:51:51,339 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 932
}

2021-12-13 17:51:51,829 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2635
}

2021-12-13 17:51:51,935 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2661
}

2021-12-13 17:51:52,046 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 932
  total_bytes: 2661
}

2021-12-13 17:51:52,153 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,265 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,381 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,489 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,599 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,704 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:52,916 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}

2021-12-13 17:51:53,446 INFO    MainThread:8932 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 2661
  total_bytes: 2661
}
local_info {
}

2021-12-13 17:51:54,533 INFO    MainThread:8932 [wandb_run.py:_append_files():2184] logging synced files
